lib/kitchen/example_element.rb

```diff
@@ -27,9 +27,15 @@ def self.short_type
     #
     # @return [ElementEnumerator]
     #
-    def titles
-      search("[data-type='title']")
+    def titles_to_rename
+      titles(except: \
+        lambda do |title|
+          title.parent.has_class?('os-caption-container') || \
+          title.parent.has_class?('os-caption') || \
+          title.parent.name == 'caption' || \
```

Comment:
This will skip over uncooked captions, but I think that'll be okay?

Reply:
yes, I think we probably don't want to impact tables when baking examples? Unless there's table-within-example-specific behavior, which I haven't seen yet